src/status_im/ui/screens/chat/components/input.cljs

```diff
@@ -2,7 +2,7 @@
   (:require [status-im.ui.components.icons.icons :as icons]
             [quo.react-native :as rn]
             [oops.core :refer [oget]]
-            [quo.react :as react]
+            [quo.react :as quo.react]
```

Comment:
nit: is this needed? I think it's usually imported as `react` elsewhere in the codebase.
imo, I prefer consistency with import names etc

Reply:
I noticed both are used in the codebase. Maybe we need to change other occurrences of importing quo.react to one or the other.

Reply:
yep, I took a look and it seems the large  majority is `react`